src/app/dashboard/_components/smart-notes/CommandMenu.tsx

```
@@ -177,102 +183,249 @@ export function CommandMenu({ onSelect, onClose, searchTerm = '', position }: Co
+        action: value.key,
+        preview: value.description,
+        type: 'block',
+        template: generateTemplate(selectedPlatform, value.key, value.description),
+      }));
+      
+      // Filter post types by search term if provided
+      const filteredPostTypes = searchTerm
+        ? postTypeCommands.filter(cmd =>
+            cmd.label.toLowerCase().includes(searchTerm.toLowerCase()) ||
+            cmd.action.toLowerCase().includes(searchTerm.toLowerCase()) ||
+            (cmd.preview && cmd.preview.toLowerCase().includes(searchTerm.toLowerCase()))
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)